presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -1009,4 +1011,17 @@ public HiveConfig setTimestampPrecision(HiveTimestampPrecision timestampPrecisio
         this.timestampPrecision = timestampPrecision;
         return this;
     }
+
+    public boolean isOptimizeSymlinkListing()
+    {
+        return this.optimizeSymlinkListing;
+    }
+
+    @Config("hive.optimize-symlink-listing")
+    @ConfigDescription("Optimize listing for SymlinkTextFormat tables with files in a single folder.")
```

Comment:
Remove period at end of description (we don't have that for others)

Let's use the word "directory" instead of "folder", since that's the more common term for Hadoop